XiEditor/AppDelegate.swift

```diff
@@ -74,7 +74,8 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             }
         case "def_style":
             if let obj = params as? [String : AnyObject] {
-                styleMap.defStyle(json: obj)
+                let font = CTFontCreateWithName("InconsolataGo" as CFString?, 14, nil)
```

Comment:
This is temporary. But if we keep the Style objects font-aware, we might re-consider whether each buffer needs its own styleMap instance instead of using a global one.